test/test-test-case.rb

```diff
@@ -65,14 +65,14 @@ def internal_backtrace_entry?(entry)
         entry.start_with?("<internal:")
       end
 
-      def normalize_location(location)
+      def normalize_backtrace(location)
```

Comment:
Could you keep `normalize_location` name?
The argument is a backtrace but we call it as (a fault's) "location" here.

Reply:
Done!